symengine/python/symengine/lib/symengine_wrapper.pyx

```diff
@@ -834,6 +834,14 @@ cdef class DenseMatrix(MatrixBase):
         deref(self.thisptr).FFLDU(deref(L.thisptr), deref(D.thisptr), deref(U.thisptr))
         return L, D, U
 
+    def jacobian(self, x):
+        cdef DenseMatrix x_ = sympify(x)
+        R = DenseMatrix(self.nrows(), x.nrows(), [0]*self.nrows()*self.nrows())
```

Comment:
Yes, I was thinking about it, that we should allow just `DenseMatrix(self.nrows(), x.nrows())`. Thanks for the catch, I was rebasing before sending a PR and I think this bug cropped in during a rebase.